build.psm1

```diff
@@ -1170,18 +1170,22 @@ function Start-PSPackage {
 
     # Build the name suffix for win-plat packages
     if ($IsWindows) {
-        $VersionTokens = $Version -split "-"
-        if ($VersionTokens.Count -gt 1) {
-            # Get the suffix like 'alpha.10-win81-x64'
-            $NameSuffix = $VersionTokens[1], $Runtime -join "-"
+        # Add the server name to the $RunTime. $runtime produced by dotnet is same for client or server
+        switch ($Runtime) {
+            'win81-x64' {$NameSuffix = 'win81-w2k12r2-x64'}
```

Comment:
I added support for win7-x86 and win7-x64 in alpha.13. Please expand your change to support those runtimes as well